cmd/cloud_sql_proxy/cloud_sql_proxy.go

```diff
@@ -493,5 +496,22 @@ func main() {
 		}),
 		Conns:              connset,
 		RefreshCfgThrottle: refreshCfgThrottle,
-	}).Run(connSrc)
+	}
+
+	signals := make(chan os.Signal, 1)
+	signal.Notify(signals, syscall.SIGTERM, syscall.SIGINT)
```

Comment:
Do you need SIGTERM too, or is SIGINT just enough? I believe the more platform-independent way is to not use the syscall package but instead lean on the os package's `os.Interrupt` (https://golang.org/pkg/os/#Signal) which is equal to syscall.SIGINT.

I checked the windows versions and `syscall.SIGINT` and `syscall.SIGTERM` both exist, but who knows what SIGTERM means there and in other places. `os.Interrupt == syscall.SIGINT` for all platforms except plan9 anyway (https://golang.org/src/os/exec_posix.go?h=Interrupt#L18) so if SIGINT is enough I say we stick with it by using `os.Interrupt` here; if you know you (or others) will want SIGTERM for sure then I think it's OK to commit and fix things later if it causes a real problem for anyone.